src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/TitanProperties.java

```diff
@@ -9,13 +9,29 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.titan.id=amazon.titan-text-express-v1
+ * aws.bedrock.model.titan.temperature=0.5 aws.bedrock.model.titan.topP=0.9
+ * aws.bedrock.model.titan.maxTokens=100 aws.bedrock.model.titan.stopSequences=Human:,
+ * Assistant:
+ *
+ * @see <a href="https://docs.ai21.com/reference/titan-complete-api-ref">...</a>
```

Comment:
This is not a valid hyperlink.  Please fix to point to correct documentation.  Consider https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan.html.